src/cmds/versions/update.js

```diff
@@ -79,11 +78,8 @@ exports.run = async function (opts) {
       is_hidden: promptResponse.is_stable ? false : !(isPublic === 'true' || promptResponse.is_hidden),
     }),
   })
-    .then(res => res.json())
-    .then(res => {
-      if (res.error) {
-        return Promise.reject(new APIError(res));
-      }
+    .then(handleRes)
```

Comment:
I'm so jealous that you got to refactor these lmao

Reply:
hahaha